test/libsolidity/syntaxTests/functionCalls/new_library.sol

```diff
@@ -0,0 +1,8 @@
+library L {}
```

Comment:
Same test as `524_accept_library_creation.sol`

Reply:
Removed the other one.